records/generator.go

```diff
@@ -66,6 +84,16 @@ type StateJSON struct {
 	Leader     string `json:"leader"`
 }
 
+// DockerNetworkSettings is a partial representation of the docker container network settings
+type DockerNetworkSettings struct {
+	IpAddress string `json:"IpAddress"`
+}
+
+// DockerInspect is a partial representation of a docker inspect output for a task's container
+type DockerInspect struct {
+	NetworkSettings DockerNetworkSettings `json:"NetworkSettings"`
+}
+
```

Comment:
We probably don't need to export these types. It'd be enough to declare them only in the scope they're required, inside `containerIPFromDockerInspect`.
```go
func containerIPFromDockerInspect(inspect string) (string, error) {
    var data []struct {
        Settings struct {
            IP string `json:"IpAddress"`
        } `json:"NetworkSettings"`
    }
    // ...
}
```